src/main/java/com/google/vinet/servlets/LoginStatusServlet.java

```diff
@@ -0,0 +1,62 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.vinet.servlets;
+
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.appengine.api.users.UserService;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that returns the login status of the user. */
+@WebServlet("/login-status")
+public class LoginStatusServlet extends HttpServlet {
+  private UserService userService;
+
+  /**
+   * Construct a LoginStatusServlet that depends on the default implementaion of UserService.
```

Comment:
implementation?

Reply:
`UserService` is an Interface I thought. `UserServiceFactory.getUserService()` returns the default implementation of the Interface.

Reply:
Sorry, I should have been clearer with my comments. I just meant that the word `implementation` is misspelled in the comment here.

Reply:
Oh I see the typo now 👍

Reply:
fixed https://github.com/googleinterns/step226-2020/pull/6/commits/6ca3fc78fb473a0e6338615ad073171c8ea74a6b